seatunnel-engine/seatunnel-engine-client/src/test/java/org/apache/seatunnel/engine/client/SeaTunnelClientTest.java

```diff
@@ -684,18 +684,55 @@ public void testGetMultiTableJobMetrics() {
                             .mapToLong(Map.Entry::getValue)
                             .sum());
             // Instantaneous rates in the same direction are directly added
-            Assertions.assertEquals(
-                    totalCount.get(SOURCE_RECEIVED_QPS),
-                    tableCount.entrySet().stream()
-                            .filter(e -> e.getKey().startsWith(SOURCE_RECEIVED_QPS + "#"))
-                            .mapToLong(Map.Entry::getValue)
-                            .sum());
-            Assertions.assertEquals(
-                    totalCount.get(SINK_WRITE_QPS),
-                    tableCount.entrySet().stream()
-                            .filter(e -> e.getKey().startsWith(SINK_WRITE_QPS + "#"))
-                            .mapToLong(Map.Entry::getValue)
-                            .sum());
+            // The size does not fluctuate more than %2 of the total value
+            Assertions.assertTrue(
+                    Math.abs(
+                                    totalCount.get(SOURCE_RECEIVED_QPS)
+                                            - tableCount.entrySet().stream()
+                                                    .filter(
+                                                            e ->
+                                                                    e.getKey()
+                                                                            .startsWith(
+                                                                                    SOURCE_RECEIVED_QPS))
```

Comment:
```suggestion
                                                                                    SOURCE_RECEIVED_QPS + "#"))
```

Reply:
We need this to filter table metrics.

Reply:
done, only writeBytes will have prefix matching error, but added for uniformity